addons/l10n_se/data/account_fiscal_position_template.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="fp_sweden" model="account.fiscal.position.template">
+            <field name="name">Sverige</field>
+            <field name="chart_template_id" ref="l10nse_chart_template" />
+        </record>
+        <record id="fp_euro" model="account.fiscal.position.template">
+            <field name="name">Europaunionen</field>
+            <field name="chart_template_id" ref="l10nse_chart_template" />
+            <field name="vat_required" eval="True" />
```

Comment:
When partner is from euro zone but not from Sweden:
- This fiscal position should be auto applied when partner has VAT number (B2B)
- fiscal position `fp_sweden` should be applied when client has not VAT (B2C).

Reply:
Correct.

Reply:
done